### PR TITLE
Add a cloud_staticfiles variable

### PR DESCRIPTION
**Pull request by jbradberry**
and make collectstatic run only once when it is true.

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -11,6 +11,7 @@ db_name: "{{ project_name }}_{{ env_name }}"
 db_user: "{{ project_name }}_{{ env_name }}"
 db_host: 'localhost'
 db_port: 5432
+cloud_staticfiles: true
```
Comment (copelco) [inline defaults/main.yml]: Should we default to ``false``?

Comment (jbradberry) [reply defaults/main.yml]: doh!  Yes, you are right.  What was I thinking?

**Comment by jbradberry** [comment]
I've now tried this with the variable set to true and set to false, and it did the expected thing both times.

**Review by copelco — APPROVED** [review]
👍

**Pull request merged**